test/test_framework.py

```diff
@@ -573,19 +573,66 @@ class CustomObject:
         # Assignment and keywords like del are not supported in lambdas so functions are used instead.
         test_funcs = [(
             lambda: setattr(obj.state, 'dict', {}),
-            lambda: self.assertEqual(dict(obj.state.dict), {})
+            lambda: self.assertEqual(obj.state.dict, {})
         ), (
             lambda: obj.state.dict.update({"a": {}}),
-            lambda: self.assertEqual(dict(obj.state.dict), {"a": {}})
+            lambda: self.assertEqual(obj.state.dict, {"a": {}})
```

Comment:
These strings should all consistently use single quotes, which is what we've been using most often for strings I believe. I don't mind much which side we go, but once we pick one we should be consistent.